---
layout: page-fullwidth
title: "Research"
meta_title: ""
subheadline: ""
teaser: ""
permalink: "/research/"
header:
    # image_fullwidth: "genvis-dna-bg_optimized_v1a.png"
---

<div data-magellan-expedition="fixed">
  <ul class="sub-nav">
    <li data-magellan-arrival="Overview"><a href="#Overview">Overview</a></li>
    <li data-magellan-arrival="Variant_interpretation"><a href="#Variant_interpretation">Variant interpretation</a></li>
    <li data-magellan-arrival="Precision_medicine"><a href="#Precision_medicine">Precision medicine</a></li>
  </ul>
</div>

<h2 data-magellan-destination="Overview">Overview</h2>
<a name="Overview"></a>

The Griffith lab is a combined research group driven by the interests of twin scientists [Malachi Griffith](http://malachigriffith.org/) and [Obi Griffith](http://obigriffith.org/). The focus of the lab is on developing methods of applied bioinformatics for personalized medicine and improved cancer care.

Our research is committed to developing open-access and open-source resources for cancer genome analysis. Research projects cover a wide spectrum of cancer informatics and clinical statistics with an emphasis on translation and application. Specifically, we use computational methods for the analysis of large cancer datasets at the molecular level (DNA, RNA, and protein) to identify markers for diagnosis, prognosis, and drug response prediction in cancer. We have contributed to the early development of methods for the analysis of transcriptional regulation (ORegAnno) and RNA-seq analysis and visualization (ALEXA-seq Platform).

















Neoantigen DNA Vaccine in Combination With Nivolumab/Ipilimumab and PROSTVAC in Metastatic Hormone-Sensitive Prostate Cancer [(NCT03532217)](https://clinicaltrials.gov/ct2/show/NCT03532217)




<h2 data-magellan-destination="Variant_interpretation">Variant interpretation</h2>
<a name="Variant_interpretation"></a>

{% include project
  title="CIViC: a community knowledgebase for expert crowdsourcing the clinical interpretation of variants in cancer"

  description="CIViC is an expert-crowdsourced knowledgebase for Clinical Interpretation of Variants in Cancer describing the therapeutic, prognostic, diagnostic and predisposing relevance of inherited and somatic variants of all types. CIViC is committed to open-source code, open-access content, public application programming interfaces (APIs) and provenance of supporting evidence to allow for the transparent creation of current and accurate variant interpretations for use in cancer precision medicine."

  team="Malachi Griffith, Nicholas Spies, Kilannin Krysiak, Josh McMichael, Adam Coffman, Arpad Danos, Benjamin Ainscough, Cody Ramirez, Lynzey Kujan, Erica Barnell, Alex Wagner, Zachary Skidmore, Connor Liu, Rachel Bilski, Robert Lesurf, Yang Yang Feng, Lee Trani, Matt Matlock, Avinash Ramu, Katie Campbell, Greg Spies, Aaron Graubert, Jason Walker, Obi Griffith"

  image="/assets/img/research/GP-127_CIViC_simplified-overview_v2d.png"

  citation="Griffith, Spies, Krysiak et al. 2017. Nature Genetics. 49(2):170–174."

  pmid="28138153"

%}

<h2 data-magellan-destination="Precision_medicine">Precision medicine</h2>
<a name="Precision_medicine"></a>

{% include project
  title="Genome analysis of relapsed adult ALL case reveals personalized therapeutic strategy"

  description="Extensive genomic analyses were performed in an adult with post-allo relapsed B-ALL. Mutations were found in EP300, NF1, IKZF1, SETD2, RB1, PAX5, NF1, ETV6, and ZNF384. Transcriptome analysis identified aberrant overexpression of the FLT3 gene. Treatment with the FLT3 inhibitor sunitinib induced a rapid clinical and molecular response. This study demonstrates a powerful proof-of-principle that comprehensive genomic studies can sometimes reveal unexpected clinically actionable therapeutic targets."

  team="Malachi Griffith, Obi Griffith, Kilannin Krysiak, Zachary Skidmore, Avinash Ramu, Alex Wagner, Katie Campbell, Robert Lesurf, Jasreet Hundal, Nicholas Spies, Benjamin Ainscough, Jason Walker"

  image="/assets/img/research/ALL1_GraphicalAbstract_neutral.png"
  citation="Griffith et al. 2016. Experimental Hematology. 44(7):603-13."

  pmid="27181063"
%}

{% include project
  title="Optimizing Cancer Genome Sequencing and Analysis"

  description="Tumors are typically sequenced to depths of 75-100x (exome) or 30-50x (whole genome). We demonstrated that current sequencing paradigms are inadequate for tumors that are impure, aneuploid or clonally heterogeneous. To reassess optimal sequencing strategies, we performed ultra-deep (up to ~312x) whole genome sequencing (WGS) and exome capture (up to ~433x) of a primary acute myeloid leukemia, its subsequent relapse, and a matched normal skin sample. We tested multiple alignment and variant calling algorithms and validated ~200,000 putative SNVs by sequencing them to depths of ~1,000x. Additional targeted sequencing provided over 10,000× coverage and ddPCR assays provided up to ~250,000x sampling of selected sites. We evaluated the effects of different library generation approaches, depth of sequencing, and analysis strategies on the ability to effectively characterize a complex tumor. This dataset, representing the most comprehensively sequenced tumor described to date, serves as an invaluable community resource."

  team="Malachi Griffith, Obi Griffith, Kilannin Krysiak, Zach Skidmore, Avinash Ramu, Jason Walker, Lee Trani"

  image="/assets/img/research/AML31_Slider_Image.png"  
  citation="Griffith, Miller et al. 2015. Cell Systems. 1(3):210-223."

  pmid="26645048"
%}

